bootstrap_cfn/stacks/iam.json

```diff
@@ -16,6 +16,7 @@
                     { "Action": [ "ec2:Describe*" ], "Resource": "*", "Effect": "Allow" },
                     { "Action": [ "rds:Describe*" ], "Resource": "*", "Effect": "Allow" },
```

Comment:
this line is where you can query the API from relevant RDS' - although makes sense to add cloudformation API as well